controllers/grafanacontactpoint_controller.go

```diff
@@ -227,6 +205,8 @@ func (r *GrafanaContactPointReconciler) buildSettings(ctx context.Context, conta
 		if err != nil {
 			return nil, fmt.Errorf("getting referenced value: %w", err)
 		}
+		r.Log.V(1).Info("overriding value", "key", override.TargetPath, "value", val)
```

Comment:
We shouldn't log the value here as it can leak sensible information

Reply:
I specifically added it back as a debug log!
It should not be logged unless the user overwrites the log level or enables development mode.